docs/TRANSLATING.md

```diff
@@ -2,13 +2,29 @@
 
 This guide is written to provide instructions from scratch for contributing to the translation of the Jamulus application to other languages.
 
-The code for Jamulus is open source, and is managed and made available via the Github site.
+The code for Jamulus is open source, and is managed and made available via the Github site. However, translators are not required to use GitHub. Jamulus is available on [Hosted Weblate](https://hosted.weblate.org/projects/jamulus/).
 
-For completeness, this document describes the use both of GitHub (using `git`) and of _Qt Linguist_.
+You have two options to translate Jamulus:
+1. Using [Weblate](https://hosted.weblate.org/projects/jamulus/jamulus-app/) (online translation, easier and more feature rich. However, as of 09/2022, it will show your email address on GitHub.) (recommended)
+2. Using git directly (more advanced, but you can test your changes on your machine locally)
 
 ---
 
-## Introduction
+## Introduction (using Weblate)
+
+For easier communication, a translator should - if possible - a GitHub account as described in the **1. Setting up** paragraph in the git section below and use it to [create an account on Hosted Weblate](https://hosted.weblate.org/accounts/register/). Ensure that the Jamulus main developers are aware of your intentions to translate to your chosen language before beginning (e.g. by [creating or commenting on a respective issue during the release process for your language on GitHub](https://github.com/jamulussoftware/jamulus/issues)). This ensures you will have proper access and support provided. You can have a look at the [adding a new language (checklist)](#adding-a-new-language-checklist) to see what to expect.
+
+## Using Weblate for translation
+
+After having logged in to Weblate - best with a linked GitHub account - visit the [Jamulus project page on Weblate](https://hosted.weblate.org/projects/jamulus/). There are several components dedicated to the Jamulus project. If you want to translate e.g. the Jamulus application, click on "Jamulus app" and choose your language. After that, translate the strings displayed on the Website. Weblate will open or update a Pull Request on the Jamulus repo in the next hours, where your translation will be reviewed.
+
+More information is provided by the [Weblate documentation](https://docs.weblate.org/en/latest/).
```

Comment:
```suggestion
More info is provided in the [Weblate documentation](https://docs.weblate.org/en/latest/).
```

Reply:
So no link to the latest version?

Reply:
@ann0see It redirects to that. The issue is more-so to avoid the "en" in cases where the browser user agent asks for a different locale.

Reply:
Ok. Thanks.